cli/args/flags.rs

```diff
@@ -2767,8 +2767,13 @@ It is especially useful for quick prototyping and checking snippets of code.
 
 TypeScript is supported, however it is not type-checked, only transpiled."
   ), UnstableArgsConfig::ResolutionAndRuntime)
-    .defer(|cmd| runtime_args(cmd, true, true)
-      .arg(check_arg(false))
+    .defer(|cmd| {
+      let cmd = compile_args_without_check_args(cmd);
```

Comment:
Instead of expanding calls from runtime_args to here, what do you think about adding an option to runtime_args() and runtime_args_parse() that excludes the `check/no-check` flags?

Reply:
Introducing a new flag for only supporting single case would be less readable in my opinion.

Also currently `runtime_args` only adds `no-check` (`check` is not added). If we introduce the flag for it, I feel what that flag does will be confusing.